pipeline_dp/private_spark.py

```diff
@@ -12,10 +13,32 @@ class PrivateRDD:
     within the specified privacy budget can be extracted from it through its API.
     """
 
-    def __init__(self, rdd, budget_accountant, privacy_id_extractor):
-        self._rdd = rdd
+    def __init__(self, rdd, budget_accountant, privacy_id_extractor=None):
+        if privacy_id_extractor:
+            self._rdd = rdd.map(lambda x: (privacy_id_extractor(x), x))
+        else:
+            self._rdd = rdd
         self._budget_accountant = budget_accountant
-        self._privacy_id_extractor = privacy_id_extractor
+
+    def map(self, fn: Callable) -> PrivateRDD:
+        """ A Spark mapValues equivalent.
```

Comment:
Nit1: """A...  (no space)  (also place fix it in other places in this file, e.g. in `__init__`)
Nit2: `mapValues` -> `map` (the user doesn't need to know that internally we keep data in tuple and do mapValues)

Reply:
fixed